ui/helpers/utils/util.js

```diff
@@ -53,26 +53,18 @@ export function formatDateWithYearContext(
     now.year === dateTime.year ? formatThisYear : fallback,
   );
 }
-/**
- * Determines if the provided chainId is a default MetaMask chain
- *
- * @param {string} chainId - chainId to check
- */
-export function isDefaultMetaMaskChain(chainId) {
-  if (
-    !chainId ||
-    chainId === CHAIN_IDS.MAINNET ||
-    chainId === CHAIN_IDS.LINEA_MAINNET ||
-    chainId === CHAIN_IDS.GOERLI ||
-    chainId === CHAIN_IDS.SEPOLIA ||
-    chainId === CHAIN_IDS.LINEA_GOERLI ||
-    chainId === CHAIN_IDS.LOCALHOST
-  ) {
-    return true;
-  }
 
-  return false;
-}
+const DEFAULT_CHAIN_IDS = new Set([
+  CHAIN_IDS.MAINNET,
+  CHAIN_IDS.LINEA_MAINNET,
+  CHAIN_IDS.GOERLI,
+  CHAIN_IDS.SEPOLIA,
+  CHAIN_IDS.LINEA_GOERLI,
+  CHAIN_IDS.LOCALHOST,
+]);
```

Comment:
It seems like there is some correlation between this list of networks and the [built-in list of networks in `shared/constants/network.ts`](https://github.com/MetaMask/metamask-extension/blob/4b38a2f0546bfef8ad2387c3946b68a55f55f48a/shared/constants/network.ts#L288). The only difference is that instead of "built-in" we're using the word "default" here. 

1. Does "default" mean the same thing?
2. If it does, what do we feel about defining an `isBuiltInChainId` function in `shared/constants/network.ts` that does something like:

    ``` typescript
    function isBuiltInChainId(chainId: string) {
      return Object.keys(BUILT_IN_NETWORKS).some((network) => {
        return BUILT_IN_NETWORKS[network].chainId === chainId;
      });
    }
    ```